public/emojis.js

```diff
@@ -684,7 +697,8 @@ const emojis = {
   WIFI: "📶",
   WINDOW: "🪟",
   WINDY: "🌬",
-  WING: "🦋",
+  WING: "🪽",
+  WINGS: "🦋",
```

Comment:
Breaking change alert! Just kidding

Reply:
I tested it out. If you had unlocked `WING` before it stays unlocked, just with a different emoji.